### DISCUSSION TITLE
Add possibility to globally include n98-magerun modules

### DISCUSSION THREAD

**Discussion by norgeindian** [wardenenv/warden#759] [Ideas]
According to https://github.com/netz98/n98-magerun2/wiki/Modules#where-can-modules-be-placed it is possible to add modules for n98-magerun.
I tried to put them into `/usr/local/share/n98-magerun2/modules` and `~/.n98-magerun2/modules`, but in both cases n98-magerun did not find them. Either I did something wrong or the folders are not synced into the container.
Would it be an idea to add them in general, so that modules can be used?

**Comment by MagePsycho** [comment]
This is great to have.
Also, I was looking for the same for globally including the bash script in all the php-fpm containers.
